ios/FluentUI/Command Bar/CommandBarButton.swift

```diff
@@ -38,12 +38,21 @@ class CommandBarButton: UIButton {
         super.init(frame: .zero)
 
         translatesAutoresizingMaskIntoConstraints = false
-        setImage(item.iconImage, for: .normal)
+
+        if #available(iOS 15.0, *) {
+            var buttonConfiguration = UIButton.Configuration.plain()
+            buttonConfiguration.image = item.iconImage
+            buttonConfiguration.contentInsets = LayoutConstants.contentInsets
+            buttonConfiguration.background.cornerRadius = 0
```

Comment:
is this necessary? i thought default was zero https://developer.apple.com/documentation/uikit/uibackgroundconfiguration/3600321-cornerradius